crates/types/src/rpc/transactions/broadcasted_invoke_transaction.rs

```diff
@@ -61,15 +50,8 @@ impl BroadcastedInvokeTransaction {
         &self,
         chain_id: Felt,
     ) -> DevnetResult<InvokeTransaction> {
-        let (entry_point_selector_field, additional_data) = if (QUERY_VERSION_OFFSET
-            + SUPPORTED_TX_VERSION)
-            == FieldElement::from(self.common.version)
-        {
-            (Felt::from(EXECUTE_ENTRY_POINT_SELECTOR.clone()), Vec::<Felt>::new())
-        } else {
-            let additional_data = vec![self.common.nonce];
-            (Felt::default(), additional_data)
-        };
+        let entry_point_selector_field = Felt::default();
```

Comment:
As discussed, let's use 1 instead of default.

Reply:
[9758041](https://github.com/0xSpaceShard/starknet-devnet-rs/pull/200/commits/9758041606dd908ad9f598550289168a0b87fecc)